Convex_hull_3/doc/Convex_hull_3/CGAL/Convex_hull_3/dual/halfspace_intersection_3.h

```diff
@@ -15,7 +15,7 @@ This version does not construct the dual points explicitely but uses a special t
 \pre The computed intersection must be a bounded convex polyhedron.
 
 \tparam PlaneIterator must be an input iterator where the value type must be `Polyhedron::Traits::Plane_3`
-\tparam Polyhedron must be a model of `ConvexHullPolyhedron_3`.
```

Comment:
`Convex_hull_3/test/Convex_hull_3/test_halfspace_intersections.cpp` should be updated to also test `Surface_mesh`.